stream-chat-android/src/main/java/com/getstream/sdk/chat/view/ReadStateView.kt

```diff
@@ -0,0 +1,120 @@
+package com.getstream.sdk.chat.view
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.Gravity
+import android.widget.RelativeLayout
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import com.getstream.sdk.chat.ChatUI.Companion.instance
+import com.getstream.sdk.chat.R
+import com.getstream.sdk.chat.utils.Utils
+import com.getstream.sdk.chat.utils.roundedImageView.CircularImageView
+import com.getstream.sdk.chat.view.messages.AvatarStyle
+import io.getstream.chat.android.client.models.ChannelUserRead
+import io.getstream.chat.android.client.models.initials
+
+public class ReadStateView : RelativeLayout {
+    private var readStateStyle: ReadStateStyle = ReadStateStyle()
+    private var avatarStyle: AvatarStyle = AvatarStyle()
+    private var isIncoming = false
+    private var reads: List<ChannelUserRead> = emptyList()
+
+    public constructor(context: Context?) : super(context) {
+        init()
+    }
+
+    public constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {
+        init()
+    }
+
+    public fun setReads(
+        reads: List<ChannelUserRead>,
+        isIncoming: Boolean,
+        readStateStyle: ReadStateStyle = ReadStateStyle(),
+        avatarStyle: AvatarStyle = AvatarStyle()
+    ) {
+        this.reads = reads
+        this.readStateStyle = readStateStyle
+        this.avatarStyle = avatarStyle
+        this.isIncoming = isIncoming
+        init()
+    }
+
+    @SuppressLint("ResourceType")
+    private fun init() {
+        removeAllViews()
+
+        if (!readStateStyle.isReadStateEnabled || reads.isEmpty()) return
+
+        val chatFonts = instance().fonts
```

Comment:
I know the Java-to-Kotlin converter does this, but let's not leave `instance()` calls in our code like this, super hard to discern